src/main/java/controller/ElementsController.java

```diff
@@ -0,0 +1,78 @@
+package controller;
+
+import javafx.collections.ObservableList;
+import javafx.scene.control.TextArea;
+import javafx.scene.control.TextField;
+
+import java.util.ArrayList;
+import java.util.Collections;
+
+/**
+ * Created by Maciek on 2017-02-10.
+ */
+public abstract class ElementsController {
+    public TextField adres_linku;
+    public TextField ilosc_wyswietlen;
+    public TextField ilosc_komentarzy;
+    public TextField ilosc_likow;
+    public TextField adres_maliny;
+    public TextArea massage;
+    public static ArrayList<TextField> requiredElements = new ArrayList<TextField>();
+    public static ArrayList<TextField> notRequiredElements = new ArrayList<TextField>();
+
+    public boolean elementsIsNotEmpty() {
+        boolean isNotEmpty = true;
+        if (requiredElements.isEmpty() || notRequiredElements.isEmpty())
+            prepareAllTextField();
+        if (isTextFieldEmpty()) {
+            isNotEmpty = false;
+            massage.setText("Wypełnij wymagane pola");
+            massage.setVisible(true);
+        }
```

Comment:
```
massage.setText("Wypełnij wymagane pola");
massage.setVisible(true);
```
to bym wywalil do metody nizej i tutaj zrobil return checkIfElementsAreEmptyAndShowErrorOnUI();